code/game/jobs/job/medical.dm

```diff
@@ -319,11 +319,12 @@
 	departments = SIMPLEDEPT(DEPARTMENT_MEDICAL)
 	department_flag = MEDSCI
 	faction = "Station"
+	alt_titles = list("Medic Trainee", "Surgical Resident")
 	total_positions = 3
 	spawn_positions = 3
 	supervisors = "the Chief Medical Officer"
 	selection_color = "#15903a"
-	access = list(access_medical, access_surgery, access_medical_equip)
+	access = list(access_medical, access_surgery, access_medical_equip, access_first_responder)
```

Comment:
i'd rather not see interns get FR access. they should be let in, if necessary.

Reply:
While I see where this is coming from, why do they get surgery access then?

Reply:
if you want to remove surgery access, feel free.

Reply:
Sir yes sir